Vagrantfile

```diff
@@ -21,6 +21,9 @@
 #     VAGRANT_NODE_BOX="debian/buster64"
 #         Specify the box to use for nodes.
 #
```

Comment:
It could be nice to add docs on:
- VAGRANT_HOSTNAME
- VAGRANT_HOSTNAME_MASTER

variables on top of the file.